### PR TITLE
Update the changelog for version 20.6.1

### PR DESCRIPTION
**Pull request by hayley-jean**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +43,7 @@ All notable changes to this project will be documented in this file.
 - Properly handle `LiveUntil` in `GrpcMessage.SendOverGrpc` and add a `Deadline` parameter [EventStore#2685](https://github.com/EventStore/EventStore/pull/2685)
 - requests with more than one url segment are correctly routed [EventStore#2691](https://github.com/EventStore/EventStore/pull/2691)
 - Handle authentication provider endpoints first followed by legacy endpoints [EventStore#2694](https://github.com/EventStore/EventStore/pull/2694)
+- Prevent clients from connecting to read only replicas which have not yet caught up [EventStore#2674](https://github.com/EventStore/EventStore/pull/2674)
```
Comment (bartelink) [inline CHANGELOG.md:46]:
```suggestion
- Prevent clients from connecting to read only replicas which have not yet caught up [EventStore#2674](https://github.com/EventStore/EventStore/pull/2674) :pray: @01100010011001010110010101110000 
```

**Review by mat-mcloughlin — DISMISSED** [review]

**Review by thefringeninja — APPROVED** [review]

**Review by thefringeninja — CHANGES_REQUESTED** [review]
What's up with the :pray: icon

**Comment by bartelink** [comment]
it conveys "thanks to the external contributor" @githubhandle  😉

**Comment by hayley-jean** [comment]
@thefringeninja I've changed the emoji to just a `Thanks to` to make it a bit clearer

**Comment by thefringeninja** [comment]
@bartelink TIL!

**Review by thefringeninja — DISMISSED** [review]

**Review by shaan1337 — COMMENTED** [review]
This line should be removed since we replaced the --disable-https flag with --insecure:
https://github.com/EventStore/EventStore/pull/2705/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR53

**Review by thefringeninja — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,11 +46,10 @@ All notable changes to this project will be documented in this file.
 - Properly handle `LiveUntil` in `GrpcMessage.SendOverGrpc` and add a `Deadline` parameter [EventStore#2685](https://github.com/EventStore/EventStore/pull/2685)
 - requests with more than one url segment are correctly routed [EventStore#2691](https://github.com/EventStore/EventStore/pull/2691)
```
Comment (alexeyzimarev) [inline CHANGELOG.md:47]: Shall we fix all capitalisation issues from the previous update?

**Review by alexeyzimarev — APPROVED** [review]

**Review by thefringeninja — APPROVED** [review]

**Pull request merged**